src/plugins/external/1-env.ts

```diff
@@ -31,7 +30,7 @@ const schema = {
   }
 };
 
-const options = {
+export const autoConfig = {
```

Comment:
Leverage `autoConfig` allows to avoid to wrap plugins in `fp` when not necessary, I didn't notice this autoloader feature.